app/templates/order/confirm_booking.hbs

```diff
@@ -52,6 +51,11 @@
       {{partial "order/goods_detail"}}
     {{/if}}
 
+    {{!-- Orders Package section --}}
+    {{#if order.isOnlineOrder }}
```

Comment:
What will happen in case of appointment? Do we want to show requests added on confirmation page?

Reply:
It is already already there.